spec/models/claim/subject_check_email_data_export_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+require "csv"
+
+RSpec.describe Claim::SubjectCheckEmailDataExport do
+  let(:submitted_student_loans_claims) do
+    Array.new(5) do
+      eligibility = create(:student_loans_eligibility, :eligible, biology_taught: true, chemistry_taught: true, computing_taught: false, languages_taught: false, physics_taught: true)
```

Comment:
I found it a bit strange that this was looping and creating 5 student loan claims with basically the same details. It makes the test data look a bit surprising. Could we not use a single claim here? We're already combining them with the Maths & Physics ones.

Reply:
Yeah, I think two claims is as good as ten here; I'll change them to one of each.